### PR TITLE
Updated layout format

### PR DESCRIPTION
**Pull request by eddieantonio**
Implements the newer `${lemma}` layout format. This obsoletes the `*.paradigm` files.

**NOTE**: the base of this PR is `new-paradigm-and-layouts` and NOT `master`!

### CODE REVIEW

File: CreeDictionary/DatabaseManager/xml_entry_lemma_finder.py [hunk 1]
```diff
@@ -103,9 +109,10 @@ def identify_entries(
         if produced_lemma not in l_to_analyses:
             produced_extra_lemmas.append(produced_lemma)
 
-    produced_extra_lemma_to_analysis: Dict[
-        FSTLemma, Set[ConcatAnalysis]
-    ] = strict_analyzer.feed_in_bulk_fast(produced_extra_lemmas)
+    produced_extra_lemma_to_analysis = cast(
+        Dict[FSTLemma, Set[ConcatAnalysis]],
+        strict_analyzer.feed_in_bulk_fast(produced_extra_lemmas),
+    )
```
Comment (Madoshakalaka) [inline CreeDictionary/DatabaseManager/xml_entry_lemma_finder.py:115]: @eddieantonio you hate the colon syntax?? 😆

**Comment by eddieantonio** [comment]
No, I like the colon syntax, but there SUDDENLY were type errors in this
file — LEGIT type errors — that somehow passed type checking
before????!??!??

Anyways, the cast is REQUIRED here! 😯

File: docs/paradigm-layouts-and-metadata.md [hunk 2]
```diff
@@ -1,90 +1,58 @@
-Paradigm layouts and metadata
-=============================
+Paradigm layouts
+================
```
Comment (Madoshakalaka) [inline docs/paradigm-layouts-and-metadata.md:2]: nice!

**Comment by Madoshakalaka** [comment]
I almost forgot you can have prs to other branches

**Pull request merged**